src/org/openjump/core/rasterimage/styler/ui/RasterStylesDialog.java

```diff
@@ -382,17 +382,50 @@ private void fixComponents() throws Exception{
         singleValuesPanel = new SingleValuesPanel(this, Utils.purgeNoData(rasterImageLayer.getRasterData(null), rasterImageLayer), rasterImageLayer);
         
         if (stats.getMin(band) == stats.getMax(band)) {
-            final DummyPanel pan = new DummyPanel("No stretched classification available, this raster has only one value: " + stats.getMax(band));
-            jTabbedPane_Type.addTab(bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.tabStretched"), pan);  
-            jTabbedPane_Type.addTab(bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.tabIntervals"), intervalPanel);
-           for (final Component c2 : intervalPanel.getComponents()) {
-              c2.setEnabled(false);
-           }
-        } else {
-            jTabbedPane_Type.addTab(bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.tabStretched"), stretchedPanel);
-            jTabbedPane_Type.addTab(bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.tabIntervals"), intervalPanel);
-        }
-        jTabbedPane_Type.addTab(bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.tabSingleValues"), singleValuesPanel);
+			final DummyPanel pan = new DummyPanel(
+					bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.message1") + " "
+							+ stats.getMax(band));
+			jTabbedPane_Type.addTab(
+					bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.tabStretched"), pan);
+			jTabbedPane_Type.addTab(
+					bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.tabIntervals"),
+					intervalPanel);
+			for (final Component c2 : intervalPanel.getComponents()) {
+				c2.setEnabled(false);
+			}
+			singleValuesPanel = new SingleValuesPanel(this,
+					Utils.purgeNoData(rasterImageLayer.getRasterData(null), rasterImageLayer), rasterImageLayer);
+			jTabbedPane_Type.addTab(
+					bundle.getString("org.openjump.core.rasterimage.styler.ui.RasterStylesDialog.tabSingleValues"),
+					singleValuesPanel);
+
+		} else if (values.length > 2048) {
```

Comment:
this does not compile. **values** is unset.

Reply:
also please add the language Bundle files to this branch. so i can have a look at the changes before committing them. then close the then superfluous PRs for them.

a PR can hold lot's of changes, please don't create one per file!